metrics.go

```diff
@@ -32,11 +33,7 @@ func (meter *OpenTelemetryMeter) Counter(name string, tags map[string]string) (g
 	meter.lock.Lock()
 	counter := meter.counterCache[key]
 	if counter == nil {
-		otCounter, err := meter.wrapped.NewInt64Counter(name)
-		if err != nil {
```

Comment:
Why the removal of the error handling?

Reply:
that's my mistake I missed error handling and now I added it again

Reply:
@chvck